solutions/cpython-tests/readme.md

```diff
@@ -0,0 +1,135 @@
+# cpython-tests
+
+This test would run cpython unit tests from 3 different versions: 3.8.11, 3.9.7 and 3.10.0 .
+
+## Skipped unit tests
+Some test files are skipped because they contain unittests that couldn't pass on Mystikos, those files are listed in `test_config/tests.failed`.
+
+For thoes test files that are listed in `test_config/tests.passed`, some of the unit tests are manually disabled by adding `@unittest.skip("Temporarily disabled in Mystikos")`, because either 1) they use unsupported feature of Mystikos (like `fork`), or 2) there are bugs pending investigation (like socket issue in `test_logging:UnixSocketHandlerTest`). To skip those unit tests, a patch file is manually generated and applied to cpython repo.
+
```

Comment:
Please specify how do we re-generate the patches with a later improved version of Mystikos.

Reply:
Added makefile command to generate diff

Reply:
That make command does not work without developers manually patching the cpython source code first. This worries me because the error-prone manual process. At least, we should have (and describe here) a way to automatically identify the failed unit tests (and the files they belong to) to assist the manual process.

Reply:
Thanks for the comment, I added detailed instructions on how to work with patch file.